com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/transaction/ZigBeeTransactionManager.java

```diff
@@ -168,14 +168,14 @@ public ZigBeeTransactionManager(ZigBeeNetworkManager manager) {
         defaultProfile = new ZigBeeTransactionProfile(NODE_RETRIES, NODE_TRANSACTIONS, NODE_DELAY);
         defaultSleepyProfile = new ZigBeeTransactionProfile(SLEEPY_RETRIES, SLEEPY_TRANSACTIONS, SLEEPY_DELAY);
 
-        defaultQueue = new ZigBeeTransactionQueue("Default");
+        defaultQueue = new ZigBeeTransactionQueue("Default", null);
```

Comment:
I think it's cleaner to have a separate constructor rather than passing in null for the address?